src/main/kotlin/org/partiql/scribe/targets/trino/TrinoExcludeTranspilation.md

```diff
@@ -0,0 +1,73 @@
+This document goes over one of the more complicated rewrites for PartiQL's `EXCLUDE` to Trino.
```

Comment:
self-review: transpilation of `EXCLUDE` to Trino is mostly the same as what we did for SparkSQL. Major difference is how to re-create the structs with excluded fields, which requires using the `CAST(ROW(...) AS ROW())` syntax.